src/sbt-test/sbt-jmh/runMain/test

```diff
@@ -1,5 +1,5 @@
 # use the given class as runner
-> runMain com.example.CustomRunnerApp -i 1 -wi 1 -f 1 .*Hello.*
+> jmh:runMain com.example.CustomRunnerApp -i 1 -wi 1 -f 1 .*Hello.*
```

Comment:
It falls out naturally since I made `Jmh` extend `Test` not to write in the same class files directory as `Compile`